### PR TITLE
ref(ui) Consolidate ? icon + tooltip pattern

### PR DESCRIPTION
**Pull request by markstory**
Consolidate and normalize the ? icon + tooltip pattern we have in a few places. I've pulled in the hover effect from one of the implementations and also made form field icons 2px smaller as 18px is not a standard icon size for us.

### CODE REVIEW

File: src/sentry/static/sentry/app/components/questionTooltip.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  className?: string;
+  size: IconSize | string;
+};
+
+const QuestionIconContainer = styled('span')<ContainerProps>`
+  display: inline-block;
+  height: ${p => p.theme.iconSizes[p.size] ?? p.size};
+  line-height: ${p => p.theme.iconSizes[p.size] ?? p.size};
+
+  & svg {
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/questionTooltip.tsx:18]: Hmm I'm guessing `IconQuestion` isn't a styled component so we can't use it as a selector?

Tooltips get portaled, so any svgs in the tooltip wrapper shouldn't be affected, and there's no way to add svgs to the content, so this should be ok.

**Review by billyvg — APPROVED** [review]
Would it be worthwhile to add a storybook item for this?

File: src/sentry/static/sentry/app/components/questionTooltip.tsx (hunk 1 above)
Comment (markstory) [reply src/sentry/static/sentry/app/components/questionTooltip.tsx:18]: Yeah tooltips will be portaled.

**Comment by markstory** [comment]
Storybook added.

File: src/sentry/static/sentry/app/views/eventsV2/tags.tsx [hunk 2]
```diff
@@ -163,21 +163,16 @@ class Tags extends React.Component<Props, State> {
...
+      <React.Fragment>
+        <StyledSectionHeading>{t('Event Tag Summary')}</StyledSectionHeading>
         {this.renderBody()}
-      </TagSection>
+      </React.Fragment>
     );
   }
 }
 
-// These styled components are used in getsentry for a paywall.
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/eventsV2/tags.tsx:174]: Is this no longer the case?

**Review by matejminar — APPROVED** [review]

File: src/sentry/static/sentry/app/views/eventsV2/tags.tsx (hunk 2 above)
Comment (markstory) [reply src/sentry/static/sentry/app/views/eventsV2/tags.tsx:174]: No. The getsentry upsell uses request stubbing now. This removed the need to rebuild the page layout in the upsell state.

**Pull request merged**